R/survival_cif.qmd

```diff
@@ -0,0 +1,171 @@
+---
+title: "Estimating Cumulative Incidence Functions Using R"
+---
+
+## Objective
+
+In this document we present how to estimate the cumulative incidence function (CIF) in R. We focus on the competing risks model where each subject experiences only one out of *k* possible events as depicted in the figure below.
+
+```{r, echo=FALSE, fig.align='center', out.width="25%"}
+knitr::include_graphics("../images/survival_competing_risks/cr.jpg")
+```
+
+## R Packages
+
+We identify three packages:
+
+-   `cmprsk` version 2.2-12, release date 2024-05-14
+
+-   `tidycmprsk` version 1.0.0, release date 2023-10-30
+
+-   `survival` version 3.7-0, release date 2024-06-01
```

Comment:
@statasaurus  - I guess we want to handle this via renv and remove the versions and release dates? What do you think?

Reply:
I realised this yesterday. I am going to update the template you made. But if you want to have session info please follow the code from [here](https://github.com/PSIAIMS/CAMIS/issues/139)